README.md

```diff
@@ -55,6 +56,13 @@ If you have a Git credential helper configured, you should not be prompted for y
 > c:/path/to/o3de/scripts/o3de register -p c:/path/to/o3de-multiplayersample
 ```
 
+#### Option #2
+If you already built the O3DE engine, use the O3DE project manager to open existing project.
+Find o3de.exe under c:/path/to/o3de/build/bin/profile/o3de.exe
+
+Click the drop down of new project button (blue), open existing... and navigate to the path to o3de multiplayer project.
```

Comment:
Suggest using the exact work usage on the button. Example:
_Click the "New Project..." drop down and select "Open Existing Project"_